rgf/rgf.py

```diff
@@ -3,12 +3,21 @@
 from glob import glob
 import numpy as np
 from sklearn.base import BaseEstimator, ClassifierMixin, RegressorMixin
-
+import platform
 ## Edit this ##################################################
 
+sys_name = platform.system()
+
 #Location of the RGF executable
-loc_exec = 'C:\\Users\\rf\\Documents\\python\\rgf1.2\\bin\\rgf.exe'
-loc_temp = 'temp/'
+WINDOWS = 'Windows'
+LINUX = 'Linux'
+
+if sys_name == WINDOWS:
+    loc_exec = 'C:\\Users\\rf\\Documents\\python\\rgf1.2\\bin\\rgf.exe'
+    loc_temp = 'temp/'
+elif sys_name == LINUX:
+    loc_exec = '/opt/rgf1.2/bin/rgf'
+    loc_temp = '/tmp'
```

Comment:
for preventing unintended loading from loc_temp  can be changed to '/tmp/rgf/' , but you variant with
storing in 'temp/' can be better and more universal. I see one problem with that approach, for example :
start ipython from root '/' directory and then this script will try to create new 'temp/' directory in '/' directory and most probably will have "permission denied" error. The normal settings for /tmp are 1777, which ls shows as drwxrwxrwt. That is: wide open, except that only the owner of a file can remove it.